meson.build

```diff
@@ -633,7 +633,8 @@ if dbusmock.returncode() != 0 and get_option('umockdev_tests').allowed()
   warning('python dbusmock not found, umockdev tests will be disabled')
 endif
 
-allow_uefi_capsule = host_machine.system() in ['linux', 'freebsd']
+allow_uefi_capsule = host_machine.system() in ['linux', 'freebsd'] and \
+                     host_machine.cpu_family() in ['x86_64', 'aarch64', 'riscv64']
```

Comment:
no x86?

Reply:
It's been disabled on x86 since fwupd was packaged for Fedora and not one person has noticed -- I don't know of any actual non-EOL x86 hardware with UEFI support. I can certainly add it if makes Debian easier or something....

Reply:
Debian currently builds fwupd-efi for `amd64`, `arm64`, `armhf`, `i386`, `riscv64`, `loong64`.

Whether those not common ones _work_; I don't really know.  I know we got changes in fwupd-efi for `loong64` explicitly.

Reply:
Okay; let's copy that then. Fixed.